addons/road-generator/nodes/road_point.gd

```diff
@@ -677,8 +677,8 @@ func update_traffic_dir(traffic_update):
 func copy_settings_from(ref_road_point: RoadPoint, copy_transform: bool = true) -> void:
 	var tmp_auto_lane = ref_road_point.auto_lanes
 	auto_lanes = false
-	lanes = ref_road_point.lanes.duplicate(true)
-	traffic_dir = ref_road_point.traffic_dir.duplicate(true)
+	lanes = ref_road_point.lanes.duplicate()
+	traffic_dir = ref_road_point.traffic_dir.duplicate()
```

Comment:
Again here, is there a reason for turning true off?

We've had bugs where the lanes array on one roadpoint ends up getting linked to another roadpoint, such that changing lanes on one changes both (which would be unexpected). I'm actually expecting to create more places where we ensure the references are not linked.